src/azure-cli/azure/cli/command_modules/vm/custom.py

```diff
@@ -2885,6 +2885,16 @@ def create_vmss(cmd, vmss_name, resource_group_name, image=None,
                                                                 build_vmss_storage_account_pool_resource,
                                                                 build_application_gateway_resource,
                                                                 build_msi_role_assignment, build_nsg_resource)
+
+    # In the latest profile, the default lb will be expected to be changed from Basic to Standard.
+    # In order to avoid breaking change which has a big impact to users,
+    # we use the hint to guide users to use Standard lb to create VMSS in the first stage.
```

Comment:
@thisisanniefang Could you help confirm that we only need to add this warning in the latest profile for public cloud?